src/components/Spreadsheet/index.js

```diff
@@ -53,12 +53,13 @@ class Spreadsheet extends Component {
      *  @return {component} FrameDataRow
      *  Callback method passed into FlatList to render a FrameDataRow for each individual move item
      */
-    renderTableRow(moveData, onMovePress) {
+    renderTableRow(moveData, onMovePress, isPortrait) {
       return (
         <FrameDataRow
           move={moveData.item}
           moveIndex={moveData.index}
           onPressHandler={() => onMovePress(moveData.item, moveData.index)}
+          isPortrait={isPortrait}
```

Comment:
Would you need isPortrait passed here, if you can just have the parent FlatList be hidden?

Saves us from passing isPortrait all over the place

Reply:
Yeah, I agree. Passing in props to grandchildren through parents where said parents don't explicitly use it is a huge code smell, and it can make it more difficult to debug issues later